File: packages/java/src/main/java/com/tesote/sdk/errors/UnprocessableContentException.java
package com.tesote.sdk.errors;

public class UnprocessableContentException extends ApiException {
    public UnprocessableContentException(String message, String errorCode, int httpStatus,
                                         String requestId, String errorId, Integer retryAfter,
                                         String responseBody, RequestSummary requestSummary,
                                         int attempts, Throwable cause) {
        super(message, errorCode, httpStatus, requestId, errorId,
                retryAfter, responseBody, requestSummary, attempts, cause);
    }
}

File: packages/csharp/src/Errors/UnprocessableContentException.cs
using System;

namespace Tesote.Sdk.Errors;

/// <summary>422 UNPROCESSABLE_CONTENT — request body was syntactically valid but semantically rejected.</summary>
public class UnprocessableContentException : ApiException
{
    /// <summary>Construct with the full required-field set.</summary>
    public UnprocessableContentException(
        string? message, string? errorCode, int httpStatus,
        string? requestId, string? errorId, int? retryAfter,
        string? responseBody, RequestSummary? requestSummary,
        int attempts, Exception? cause)
        : base(message, errorCode, httpStatus, requestId, errorId,
               retryAfter, responseBody, requestSummary, attempts, cause)
    {
    }
}
